src/core_plugins/kibana/public/management/sections/objects/components/objects_table/__tests__/objects_table.test.js

```diff
@@ -425,7 +425,6 @@ describe('ObjectsTable', () => {
       expect(mockSavedObjectsClient.delete).toHaveBeenCalledWith(mockSavedObjects[0].type, mockSavedObjects[0].id);
       expect(mockSavedObjectsClient.delete).toHaveBeenCalledWith(mockSavedObjects[1].type, mockSavedObjects[1].id);
       expect(component.state('selectedSavedObjects').length).toBe(0);
-      expect(defaultProps.savedObjectsClient.find.mock.calls.length).toBe(2);
```

Comment:
is this meant to be removed? does its equivalence need to be added elsewhere or in another test?